python/hopsworks_common/kafka_topic.py

```diff
@@ -14,19 +14,25 @@
 #   limitations under the License.
 #
 
+from __future__ import annotations
+
 import json
+from typing import Optional
 
 import humps
 from hopsworks_common import usage, util
+from hopsworks_common.constants import KAFKA_TOPIC
 from hopsworks_common.core import kafka_api
 
 
 class KafkaTopic:
+    """Configuration for a Kafka topic."""
+
     def __init__(
         self,
-        name=None,
-        num_of_replicas=None,
-        num_of_partitions=None,
+        name: Optional[str] = KAFKA_TOPIC.CREATE,
```

Comment:
Why changing the default to "CREATE" rather than keeping None?

Reply:
It was this way in hsml: https://github.com/logicalclocks/hopsworks-api/blob/edeffbc4169ac6ed589235a11ac873158fbe9d93/python/hsml/kafka_topic.py#L29